### PR TITLE
Add field for last GC pause time

### PR DESCRIPTION
**Pull request by tevjef**
### Required for all PRs:

- [x] CHANGELOG.md updated (we recommend not updating this until the PR has been approved by a maintainer)
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [x] README.md updated (if adding a new plugin)

### CODE REVIEW

File: plugins/inputs/influxdb/influxdb.go [hunk 1]
```diff
@@ -202,6 +203,7 @@ func (i *InfluxDB) gatherURL(
 						"next_gc":         m.NextGC,
 						"last_gc":         m.LastGC,
 						"pause_total_ns":  m.PauseTotalNs,
+						"pause_ns":        m.PauseNs[(m.NumGC+255)%256],
```
Comment (sparrc) [inline plugins/inputs/influxdb/influxdb.go]: what if NumGC == 0? Should the index be 255?

Comment (tevjef) [inline plugins/inputs/influxdb/influxdb.go]: The index doesn't matter when NumGC == 0 since every element in PauseNs[0:256] will be zero.

**Comment by tevjef** [comment]
@sparrc Any reason why this wouldn't make the 1.2.0 release?

**Comment by sparrc** [comment]
None in particular, just a large backlog :), I'll try to get it in since it's a rather small change

**Pull request merged**